specs/ceph_crud_flows.adoc

```diff
@@ -0,0 +1,557 @@
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          name: "Create Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.create.Create
+          type: Create
+          uuid: bd0155a8-ff15-42ff-9c76-5176f53c13e0
+        Delete:
+          enabled: true
+          help: "Pool delete Atom"
+          inputs:
+            mandatory:
+              - Pool.pool_id
+          name: "Delete Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.delete.Delete
+          type: Delete
+          uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3720
+        Update:
+          enabled: true
+          help: "Pool update Atom"
+          inputs:
+            mandatory:
+              - Pool.pool_id
+            optional:
+              - Pool.poolname
+              - Pool.size
+              - Pool.pg_num
+              - Pool.quota_enabled
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          name: "Update Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.update.Update
+          type: Update
+          uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3721
+      flows:
+        DeletePool:
+          atoms:
+            - tendrl.ceph_integration.objects.Pool.atoms.delete.Delete
+          help: "Delete Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+              - TendrlContext.sds_name
+              - TendrlContext.sds_version
+              - TendrlContext.integration_id
+          run: tendrl.ceph_integration.objects.Pool.flows.delete_pool.DeletePool
+          type: Delete
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3b9
+        UpdatePool:
+          atoms:
+            - tendrl.ceph_integration.objects.Pool.atoms.update.Update
+          help: "Update Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+            optional:
+              - Pool.poolname
+              - Pool.size
+              - Pool.pg_num
+              - Pool.quota_enabled
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          run: tendrl.ceph_integration.objects.Pool.flows.update_pool.UpdatePool
+          type: Update
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3b2
+      attrs:
+        crush_ruleset:
+          help: "The ID of a CRUSH ruleset to use for this pool. The specified ruleset must exist."
+          type: Integer
+        erasure_code_profile:
+          help: "For erasure pools only.It must be an existing profile "
+          type: String
+        min_size:
+          help: "sets the number of replicas for objects in the pool"
+          type: Integer
+        pg_num:
+          help: "The total number of placement groups for placement purposes."
+          type: Integer
+        pgp_num:
+          help: "The total number of placement groups for the pool."
+          type: Integer
```

Comment:
pgp_num is actually set the same value that of pg_num. Its present here only for listing purpose.